lib/config/fileconf.go

```diff
@@ -2081,11 +2081,11 @@ type AppAWS struct {
 // PortRange can be used to describe a single port in which case the Port field is the port and the
 // EndPort field is 0.
 type PortRange struct {
-	// Port describes the start of the range. It must be between 1-65535.
-	Port uint32 `yaml:"port"`
-	// EndPort describes the end of the range, inclusive. It must be between 2-65535 and be greater
-	// than Port when describing a port range. When describing a single port, it must be set to 0.
-	EndPort uint32 `yaml:"end_port,omitempty"`
+	// Port describes the start of the range. It must be greater than 0.
+	Port uint16 `yaml:"port"`
```

Comment:
Does this change make sense? In protos, we're forced to use uint32. But everywhere else in the code I figured I should just use uint16 since it's closer to the range of valid values.

Reply:
I think it would be fine to use plain `int`.
We have bounds checks, so I think either way it's fine 👍